tensorflow/tools/ci_build/windows/gpu/pip/build_tf_windows.sh

```diff
@@ -46,7 +46,7 @@ run_configure_for_gpu_build
 
 clean_output_base
 
-bazel build -c opt --config=win-cuda $BUILD_OPTS tensorflow/tools/pip_package:build_pip_package || exit $?
+bazel build -c opt $BUILD_OPTS tensorflow/tools/pip_package:build_pip_package || exit $?
```

Comment:
Yes, because `--config=win-cuda` is added into `.tf_configure.bazelrc` file when running ./configure with gpu support.
See https://github.com/tensorflow/tensorflow/blob/a4a3a3335bd273e6224d16a934b59531ef239893/configure.py#L726